spec/setup_acceptance_node.pp

```diff
@@ -20,4 +26,17 @@
   package { 'glibc-langpack-en':
     ensure => installed,
   }
+
+  yumrepo { 'katello_staging':
```

Comment:
Required work around since these packages needed are not in the nightly repository just yet, a chicken and egg problem where these tests need nightly to get released but nightly needs the fixes here to release. I added the tests and caveats to show that this change does work on El8.